loadtest/user/userentity/actions.go

```diff
@@ -881,3 +884,71 @@ func (ue *UserEntity) SetCurrentChannel(channel *model.Channel) error {
 func (ue *UserEntity) ClearUserData() {
 	ue.store.Clear()
 }
+
+// GetLogs fetches the logs.
+func (ue *UserEntity) GetLogs(page, perPage int) error {
+	_, resp := ue.client.GetLogs(page, perPage)
+	if resp.Error != nil {
+		return resp.Error
+	}
+	return nil
+}
+
+// GetAnalytics fetches the system analytics.
+func (ue *UserEntity) GetAnalytics() error {
+	teamIds, err := ue.GetTeamsForUser(ue.store.Id())
+	if err != nil {
+		return err
+	}
+	if len(teamIds) == 0 {
+		return fmt.Errorf("user does not have any teams")
+	}
+	_, resp := ue.client.GetAnalyticsOld("", teamIds[0])
```

Comment:
Is there any reason to pick first team every time?

Reply:
Just keeping it simple. This is not a load test in any sense. Just a basic sanity test of the cluster APIs.